### PR TITLE
Bug 994430 - Improve directory tree style

### PR DESCRIPTION
**Pull request by stephaniehobson**
Moved from image solution to CSS only solution and added some left padding.

Testing:
https://developer.mozilla.org/en-US/docs/User:wbamberg/directory-tree (and in RTL)

### CODE REVIEW

File: kuma/static/styles/components/content.styl [hunk 1]
```diff
@@ -154,24 +154,46 @@ $table-blue = #d4dde4;
...
-        }
-    }
+            position: relative;
+
+            &:before,
+            &:after {
+                content: '';
+                position: absolute;
+                bidi-style(left, -15px, right, auto);
+                display: block;
```
Comment (willkg) [inline kuma/static/styles/components/content.styl]: Is there a style rule in MDN for having CSS properties be alphabetical?

**Comment by willkg** [comment]
Other than that question, I think this looks great! r+

**Comment by stephaniehobson** [comment]
Alphabetical properties are the only part of the mozorg coding guidelines we don't follow.

**Comment by willkg** [comment]
Good to know. Merging!

**Pull request merged**